recipe/bld.bat

```diff
@@ -0,0 +1,26 @@
+pushd "%SRC_DIR%" || exit /b
+
+rem Inline "installzlib.bat" and modify so it can be executed without MSYS
+curl -O https://zlib.net/zlib1211.zip || exit /b
```

Comment:
Can we add this as another win-only source with an expected hash (and target folder) under the `source:` section of `meta.yaml`?

Reply:
Nice thought, this would also allow removing the dependency on `curl` and `unzip` I guess... I'll try, thx!

Reply:
Fixed in 6c9cb73